healthtools/scrapers/base_scraper.py

```diff
@@ -225,43 +229,41 @@ def archive_data(self, payload):
         '''
         Upload scraped data to AWS S3
         '''
-        data_key = DATA_DIR + self.data_key
-        data_archive_key = DATA_DIR + self.data_archive_key
```

Comment:
@DavidLemayian so I leave this as is 😕 ??..from my tests and scraper, this runs correctly